index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Landify Project</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
```

Comment:
- Add the font used here (inter) from google fonts

Reply:
okay , you can check, i have added only SELECT EXTRALIGHT 200 and Regular 400 https://prnt.sc/uUL38cb84Xg-